source/slang/hlsl.meta.slang

```diff
@@ -108,7 +108,33 @@ struct ByteAddressBuffer
         {
         case hlsl: __intrinsic_asm ".Load";
         default:
-            return __byteAddressBufferLoad<uint>(this, location);
+            return __byteAddressBufferLoad<uint>(this, location, 0);
+        }
+    }
+
+    [__readNone]
+    [ForceInline]
+    [require(cpp_cuda_glsl_hlsl_metal_spirv, byteaddressbuffer)]
+    uint Load(int location, int alignment)
```

Comment:
This overload is not necessary. No alignment needed for loading a single uint value.
We only need Load(int location, int alignment) for the generic version `Load<T>`.